.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: generic
+
+sudo: false
+
+matrix:
+  include:
+    - { os: linux, env: PYTHON_VERSION=2.7 }
```

Comment:
While that's probably fine, we might need to address some import issues on python2.7. We might need to add some `from __future__ import absolute_import` in affected modules.